app/controllers/api/v1/beaches_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class BeachesController < ExternalApiController
+      def index
+        return forbidden unless current_user.beach_admin?
```

Comment:
maybe use `any_admin?`, as I see no reason to not allow regular admins to check this.